apps/marketplace/pages/ResetPasswordPage.js

```diff
@@ -43,10 +36,27 @@ export class ResetPasswordPageComponent extends Component {
     })
   }
 
+  getTokenFromURL() {
+    const token = this.props.location.pathname.substring(
+      this.props.match.url.length + 1,
+      this.props.location.pathname.length
+    )
+    return token
+  }
+
+  getEmailFromQueryString() {
+    const parsed = parse(this.props.location.search.substr(1))
+    let emailAddress = ''
+    if (parsed.e) {
+      emailAddress = parsed.e
+    }
+    return emailAddress
+  }
```

Comment:
Should we move these two functions, `getTokenFromURL ` and `getEmailFromQueryString `, into a helper file since they're also used by the `CreateUserPage` component?